Source/Widgets/Viewer/Viewer.js

```diff
@@ -176,6 +180,55 @@ define([
         });
     }
 
+    function enableVRUI(viewer, enabled) {
+        var geocoder = viewer._geocoder;
+        var homeButton = viewer._homeButton;
+        var sceneModePicker = viewer._sceneModePicker;
+        var baseLayerPicker = viewer._baseLayerPicker;
+        var animation = viewer._animation;
+        var timeline = viewer._timeline;
+        var fullscreenButton = viewer._fullscreenButton;
+        var infoBox = viewer._infoBox;
+        var selectionIndicator = viewer._selectionIndicator;
+
+        var display = enabled ? 'none' : 'block';
```

Comment:
I keep forgetting about the two versions thing.  I guess it's better to just shut it off for now.  I would love to see a world-space based selection indicator in the future anyway, which will fix the issue overall.